src/target/riscv/riscv-013.c

```diff
@@ -1097,15 +1155,12 @@ static int examine(struct target *target)
 			r->xlen[i] = 64;
 		}
 
-		LOG_DEBUG("hart %d has XLEN=%d", i, r->xlen[i]);
```

Comment:
I guess what I'm saying before the trigger enumeration, since that takes a long time in sim. It would be ncie to see that it at least identified the XLEN before going into that.